addons/main/script_macros.hpp

```diff
@@ -37,6 +37,8 @@
 #define GETGVAR(var1,var2) GETMVAR(GVAR(var1),var2)
 #define GETEGVAR(var1,var2,var3) GETMVAR(EGVAR(var1,var2),var3)
 
+#define QGETGVAR(var1,var2) QUOTE(GETMVAR(QGVAR(var1),var2))
```

Comment:
QGETMVAR

Reply:
Im not sure if I agree with it, the not quoted one is 2 lines higher